Makefile

```diff
@@ -1,8 +1,6 @@
 .PHONY: help
 help:
 	@echo "make help              Show this help message"
-	@echo 'make services          Run the services that `make dev` requires'
-	@echo "                       (Postgres, Elasticsearch, etc) in Docker Compose"
```

Comment:
Although you can still call `make services` directly you never need to, so I removed it from the docs.